### DISCUSSION TITLE
WARNING: The system cannot find the file specified. (os error 2): quarto-preview.js

### DISCUSSION THREAD

**Discussion by Harbour-N** [quarto-dev/quarto-cli#9789] [Q&A] [answered]
### Description

I just downloaded the most recent version of quarto. Now when I try to preview quarto docs I get the warning:

` WARNING: The system cannot find the file specified. (os error 2): readfile 'C:\Users\nicho\AppData\Local\Programs\Quarto\share\preview\quarto-preview.js' `

What is causing this warning? The preview itself still works fine.

**Comment by mcanouil** [comment, marked as answer]
See:

- https://github.com/quarto-dev/quarto-cli/issues/9574
